flask_azure_oauth/resource_protector.py

```diff
@@ -14,16 +14,15 @@ class ResourceProtector(_ResourceProtector):
     - overloading the 'raise_error_response' method to catch exceptions as API errors and return them to the client
     """
 
-    @classmethod
-    def deregister_token_validator(cls, token_validator: BearerTokenValidator) -> None:
+    def deregister_token_validator(self, token_validator: BearerTokenValidator) -> None:
         """
         This method adds a counterpart to the 'register_token_validator' in the 'ResourceProtector' class to allow a
         previously registered token validator type to be removed
 
         :type token_validator: BearerTokenValidator
         :param token_validator: Previously registered token validator
         """
-        del cls.TOKEN_VALIDATORS[token_validator.TOKEN_TYPE]
+        del self._token_validators[token_validator.TOKEN_TYPE]
```

Comment:
Addresses this commit:
[Move token validator registry into instance.](https://github.com/lepture/authlib/commit/b8c2c29788a9394ae31dd267bc770fc891c009f7)

Reply:
Is there a reason why you want to `deregister_token_validator`?

I think this method is created due to Token Validators are registered into class variables. But now, token validators are registered into instance, there seems no need to deregister them.